### PR TITLE
feat: use common template module code from common dev assets

### PR DESCRIPTION
**Pull request by akocbek**
### Description

Using common template module code from common-dev-assets

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [x] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ Follow this process to create and submit a Terraform module.
 
 1.  Create a repository from this repository template by clicking `Use this template` in the upper right of the GitHub UI.
 
-    For more information about creating a repository from a template, see the [GitHub docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
-1.  Select `terraform-ibm-modules` as the owner.
-1.  Enter a name for the module in format `terraform-ibm-<NAME>`, where `<NAME>` reflects the type of infrastructure that the module manages.
+> For more information about creating a repository from a template, see the [GitHub docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
```
Comment (SirSpidey) [inline README.md:35]: Would using hex or an entity work to preserve whitespace? As in, 

```
1.  Create a repository from this repository template by clicking `Use this template` in the upper right of the GitHub UI.

\xA0 \xA0 \xA0 \xA0For more information about creating a repository from a template, see the [GitHub docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).

or
&nbsp; &nbsp; &nbsp; &nbsp; For more information...
```

File: README.md [hunk 2]
```diff
@@ -51,23 +53,24 @@ Implement the logic for your module by updating the `main.tf`, `version.tf`, `va
...
 
 ### Update the content in the readme file
 
 After you implement the logic for your module and create examples and tests, update this readme file in your repository by following these steps:
 
 1.  Update the title heading and add a description about your module.
-1.  Update the badge links.
-1.  Remove all the content in this H2 heading section.
-1.  Complete the [Usage](#usage), [Required IAM access policies](#required-iam-access-policies), and [Examples](#examples) sections. The [Requirements](#requirements) section is populated by a pre-commit hook.
+2.  Update the badge links.
```
Comment (SirSpidey) [inline README.md:63]: Is the numbering not preserved even when the list is not indented? It's easier to maintain if every list item starts with `1. `.

File: README.md [hunk 3]
```diff
@@ -101,12 +104,12 @@ Manage > Access (IAM) > Access groups > Access policies.
...
-        - `Editor` platform access
-        - `Manager` service access
+- **Sample Account Service** service
+- `Editor` platform access
+- `Manager` service access
 - IAM Services
-    - **Sample Cloud Service** service
-        - `Administrator` platform access
+- **Sample Cloud Service** service
+- `Administrator` platform access
```
Comment (SirSpidey) [inline README.md:112]: Somehow we need to preserve the indentation so that contributors can copy the list with nested items.

File: README.md [hunk 4]
```diff
@@ -92,7 +95,7 @@ unless real values don't help users know what to change.
 <!-- PERMISSIONS REQUIRED TO RUN MODULE
 If this module requires permissions, uncomment the following block and update
 the sample permissions, following the format.
-Replace the sample Account and IBM Cloud service names and roles with the
+Replace the sample Account and Cloud service names and roles with the
```
Comment (SirSpidey) [inline README.md:98]: We should keep the IBM here in the source

**Review by SirSpidey — COMMENTED** [review]
This indentation issue issues seem like a pretty significant bug. Looks like even whitespace and indentation isn’t preserved even in codeblocks ([issue](https://github.com/terraform-docs/terraform-docs/issues/577#issuecomment-1092895293)). 

I'm impressed with the <blockquote> (`>`) workaround for separate paragraphs in list items, but it’s not accurate output (it’s not a quotation).

The issue identifies using an include as a workaround, but that’s a writer’s maintenance nightmare.

I wondered in one of my comments whether using hex or an entity might work to preserve whitespace? As in, `\xA0` or `nbsp;`

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**